draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -463,16 +463,49 @@ in the `Cookie` header field.
 
 If the user agent receives a new cookie with the same cookie-name,
 domain-value, and path-value as a cookie that it has already stored, the
-existing cookie is evicted and replaced with the new cookie. Notice that
-servers can delete cookies by sending the user agent a new cookie with an
-Expires attribute with a value in the past.
+existing cookie is evicted and replaced with the new cookie.
 
 Unless the cookie's attributes indicate otherwise, the cookie is returned only
 to the origin server (and not, for example, to any subdomains), and it expires
 at the end of the current session (as defined by the user agent). User agents
 ignore unrecognized cookie attributes (but not the entire cookie).
 
 
+### Cookie deletion
+
+A cookie that has previously been saved on the user agent can be deleted from
```

Comment:
Can you indent this to align with the rest of the doc?

Reply:
Updated section, but it's not showing (to me) as indented differently to the rest of the document...